app/timetables/admin.py

```diff
@@ -80,5 +80,5 @@ class VendorAdmin(DefaultAdmin):
 
 admin.site.empty_value_display = ''
 
-other_models = [Admin]
+other_models = [TimetableManagement]
```

Comment:
Why are we not registering the `TimetableManagement` model directly? Why are we creating an unneeded list here? `admin.site.register(TimetableManagement)` should work just fine.